### PR TITLE
feat: add security insights plugin

### PR DESCRIPTION
**Pull request by lewtakm**
PR adds security insights plugin into Roadie.io website.

### CODE REVIEW

File: content/plugins/descriptions/security-insights.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        &lt;EntityPageLayout>
+          &lt;EntityPageLayout.Content
+            path="/security-insights"
+            title="Security Insights"
+            element={&lt;SecurityInsightsRouter entity={entity} />}
+          />
+          &lt;/EntityPageLayout>
+        &lt;/EntityPageLayout>
+      )
+  - intro: Run backstage app with and navigate to services tabs.
```
Comment (martina-if) [inline content/plugins/descriptions/security-insights.yaml:60]: Typo here? `Run backstage app with and...` something is missing

File: content/plugins/descriptions/security-insights.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    language: bash
+    code: |
+      yarn start
+  - intro: Widgets setup.
+  - intro: You must install plugin by following the steps above to add widget to your Overview.
```
Comment (martina-if) [inline content/plugins/descriptions/security-insights.yaml:65]: nitpick:

```suggestion
  - intro: You must install this plugin by following the steps above to add the widget to your Overview.
```

File: content/plugins/descriptions/security-insights.yaml (hunk 1 above)
Comment (martina-if) [reply content/plugins/descriptions/security-insights.yaml:60]: Or should it be "Run the backstage app with the following command and navigate to the services tab"?

**Comment by lewtakm** [comment]
@martina-if  thanks for a review, to make plugins descriptions consistent I updated two more to be the same.

**Review by martina-if — APPROVED** [review]

**Pull request merged**